Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -271,13 +271,21 @@ struct PaymentSheetButtons: View {
                     }.padding(.horizontal)
                     HStack {
                         if let psfc = playgroundController.paymentSheetFlowController {
-                            Button {
-                                psFCOptionsIsPresented = true
-                            } label: {
-                                PaymentOptionView(paymentOptionDisplayData: playgroundController.paymentSheetFlowController?.paymentOption)
+                            VStack {
+                                Button {
+                                    psFCOptionsIsPresented = true
+                                } label: {
+                                    PaymentOptionView(paymentOptionDisplayData: playgroundController.paymentSheetFlowController?.paymentOption)
+                                }
+                                .disabled(playgroundController.paymentSheetFlowController == nil)
+                                Button {
+                                    psFCOptionsIsPresented = true
+                                } label: {
+                                    PaymentOptionInfoView(paymentOptionDisplayData: playgroundController.paymentSheetFlowController?.paymentOption)
+                                }
```

Comment:
Why is this shown twice?  Can you post a screenshot

Reply:
Twice?  `PaymentOptionView` shows the image and label, and `PaymentOptionInfoView` shows the paymentMethodType and billingDetails.

Initially, i tried putting everything into `PaymentOptionView`, but it broke a number of unit tests due to all of the accessibility identifiers with billingDetails.  On second thought, we should just remove all of the accessibility tags in BillingDetailsView, which should resolve the issue i was running into.  I will post a screenshot after i've made these changes.

Reply:
Ah I didn't see it was `PaymentOptionInfoView` vs `PaymentOptionView`.  Proposed change sounds good.

Reply:
![Screenshot 2024-01-11 at 2 24 42 PM](https://github.com/stripe/stripe-ios/assets/99628984/1f925d2c-f826-432d-ab4e-88a2b376677d)